dotcom-rendering/src/components/LiveBlockContainer.tsx

```diff
@@ -115,6 +108,22 @@ const BlockByline = ({
 	);
 };
 
+const liveBlockContainerStyles = (isPinnedPost: boolean) => css`
+	padding: ${space[2]}px ${SIDE_MARGIN_MOBILE}px;
+	box-sizing: border-box;
+	margin-bottom: ${space[3]}px;
+	background: ${palette('--live-block-container-background')};
+	${!isPinnedPost &&
+	`
+		border-top: 1px solid ${palette('--live-block-border-top')};
+		border-bottom: 1px solid ${palette('--live-block-border-bottom')};
+	`}
```

Comment:
Appreciate this is preexisting so might be a separate refactor but I think the preferred style in DCR is to make styles decisions outside of the css template literate eg 
```
const liveBlockContainerStyles = css`
	padding: ${space[2]}px ${SIDE_MARGIN_MOBILE}px;
	box-sizing: border-box;
	margin-bottom: ${space[3]}px;
	background: ${palette('--live-block-container-background')};
	${from.tablet} {
		padding: ${space[2]}px ${SIDE_MARGIN}px;
		padding-left: ${LEFT_MARGIN_DESKTOP}px;
	}
	
const liveBlockBorderStyles	css`
		border-top: 1px solid ${palette('--live-block-border-top')};
		border-bottom: 1px solid ${palette('--live-block-border-bottom')};
	`}
`;
```

and then compose styles in the css prop eg
```
			css={[liveBlockContainerStyles, !IsPinnedPost && liveBlockBorderStyles]}
```

(There's a term for this pattern but I can't remember it!)